### ISSUE TITLE
pgcenter - Perl hash contains nonexistent column 

### ISSUE THREAD

**Issue by ishibaev** [lesovsky/pgcenter#102]
**Describe the bug**
postgres version 
pgcenter installed into db postgres
several views return error:
```
postgres=# select * from pgcenter.sys_proc_netdev limit 1;
ERROR:  Perl hash contains nonexistent column "col20"
CONTEXT:  PL/Perl function "get_proc_stats"
postgres=# select * from pgcenter.sys_proc_stat limit 1;
ERROR:  Perl hash contains nonexistent column "col11"
CONTEXT:  PL/Perl function "get_proc_stats"
postgres=# select * from pgcenter.sys_proc_meminfo limit 1;
ERROR:  Perl hash contains nonexistent column "col5"
```
**Environment**

Describe the environment where the bug occurred.
- OS: Red Hat Enterprise Linux Server release 7.9 (Maipo)
- pgCenter Version 0.9.1 (also in 0.9.0)
- pgCenter installation method: tar-ball
- PostgreSQL Version -  PostgreSQL 9.6.19 on x86_64-redhat-linux-gnu, compiled by gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-37), 64-bit
- Did you try build pgCenter from master branch and reproduce the issue? no, we cant

**To Reproduce**
install pgcenter into postgres db & run ' ./pgcenter top -d postgres -h XXX':
```
ERROR: Perl hash contains nonexistent column "col5" (SQLSTATE 42703)
```
**Expected behavior**
expect 'top' output

**Additional context**
- PostgreSQL error logs
```
SELECT ERROR:  Perl hash contains nonexistent column "col5"
SELECT CONTEXT:  PL/Perl function "get_proc_stats"

SELECT STATEMENT:  
SELECT metric, metric_value
                        FROM pgcenter.sys_proc_meminfo
                        WHERE metric IN ('MemTotal:','MemFree:','SwapTotal:','SwapFree:', 'Cached:','Dirty:','Writeback:','Buffers:','Slab:')
                        ORDER BY 1;
```

**Comment by lesovsky** [comment]
Seems different versions of Perl split strings in different way. The issue is reproduced with Perl 5.16.3 (Centos 7.9.2009) and is not reproduced with newer Perl 5.30.0 (Ubuntu 20.04).

The plperl function has been fixed in [v0.9.2](https://github.com/lesovsky/pgcenter/releases/tag/v0.9.2), try to reproduce the issue using v0.9.2.

**Comment by ishibaev** [comment]
hello! thanks for quick response!
it works!
